datatypes/src/collections/multi_point_collection.rs

```diff
@@ -100,6 +100,44 @@ impl GeoFeatureCollectionRowBuilder<MultiPoint> for FeatureCollectionRowBuilder<
     }
 }
 
+impl MultiPointCollection {
+    pub fn coordinates(&self) -> &[Coordinate2D] {
```

Comment:
Maybe this method can be part of FeatureCollection?

Reply:
how? the nesting is different for point/lines/polygons...

Reply:
maybe add trait for all geom types and use this in collections?

Reply:
a trait would be possible, but is out of scope of this PR. Not sure where we would need direct access to the coordinates of a multipolygoncollection eg. without infos about the rings, other than in tests.

Reply:
For the OpenCL implementation of lines and polygons that is not part of this PR ;-).

Reply:
then this should be addressed later.